build/artifacts.props

```diff
@@ -112,6 +112,7 @@
     <PackageArtifact Include="Microsoft.AspNetCore.Proxy" Category="noship" />
     <PackageArtifact Include="Microsoft.AspNetCore.RangeHelper.Sources" Category="noship" />
     <PackageArtifact Include="Microsoft.AspNetCore.Razor" Category="ship" Metapackage="true" RuntimeStore="true"/>
+    <PackageArtifact Include="Microsoft.AspNetCore.Razor.Design" Category="ship" Metapackage="true" RuntimeStore="true"/>
```

Comment:
@JunTaoLuo @natemcmaster FYI for potential ASP.NET platform and/or ProdCon impact

Reply:
Shouldn't affect prodcon unless this adds some kind of new build dependency (only builds in VS, or something)